js/bookmark.js

```diff
@@ -0,0 +1,26 @@
+// here it targets all bookmarks
+//bookmark.js
+
+function BookmarkToggle() {
+  const sections = document.querySelectorAll("section");
```

Comment:
ich glaube, das brauchst du nicht. ich finde die verschachtelung der for each schleifen ziemlich kompliziert. ich bin nicht sicher, aber vllt könntest du den code auch so schreiben? also ohne die sections und dann mit der definition von links außerhalb der schleife?


```suggestion
  const sections = document.querySelectorAll("section");
```

function BookmarkToggle() {
  const links = document.querySelectorAll("section a.test > i");

  links.forEach((link) => {
    link.addEventListener("click", () => {
      link.classList.toggle("card__icon__bookmarked");
    });
  });
}

Reply:
Das hat nicht geklappt, wie es sollte. Ich versuch's noch mal. Bin noch nicht so geübt. 😬